remote_monitoring_demo/recommendations_Infra_demo/recommendations_demo.sh

```diff
@@ -0,0 +1,440 @@
+#!/bin/bash
+#
+# Copyright (c) 2023, 2023 Red Hat, IBM Corporation and others.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+current_dir="$(dirname "$0")"
+source ${current_dir}/../../common_helper.sh
+source ${current_dir}/recommendations_demo/recommendation_helper.sh
+# Default docker image repos
+AUTOTUNE_DOCKER_REPO="docker.io/kruize/autotune_operator"
+
+echo "${current_dir}/recommendations_demo/recommendation_helper.sh"
+
+# Default cluster
+CLUSTER_TYPE="openshift"
```

Comment:
@kusumachalasani - Running this on openshift fails with the below error, please fix this.

```
./recommendations_demo.sh -c openshift  -r --validate
```

ERROR: Please install minikube and try again
       Minikube resource config needed for demo:
       CPUs=8, Memory=16384MB

Reply:
Done.